app/models/role.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Role class
+class Role < ApplicationRecord
```

Comment:
where the roles are created? I assume that when a new movie is created and persons are to it as "janitor man" of instance. Does this logic exist?